java/com/trainning/framgia/mvp/screen/login/LoginActivity.java

```diff
@@ -0,0 +1,55 @@
+package com.trainning.framgia.mvp.screen.login;
+
+import android.view.View;
+import android.widget.EditText;
+import android.widget.Toast;
+import com.trainning.framgia.mvp.R;
+import com.trainning.framgia.mvp.data.User;
+import com.trainning.framgia.mvp.screen.BaseActivity;
+
+public class LoginActivity extends BaseActivity
+        implements View.OnClickListener, LoginContract.View {
+    private EditText mEditEmail, mEditPass;
+    private LoginPresenter mPresenter;
```

Comment:
e nên khai báo contract,
thì interface contract mới có giá trị
```suggestion
    private LoginContract.Presenter mPresenter;
```